### ISSUE TITLE
Go on-demand binary server

### ISSUE THREAD

**Issue by tj** [tj/sponsors#15]
Me and @caarlos0 spoke about this a while back, no idea if anyone got to it, but I still think it would be great to have something like:

```
$ curl -sf https://bins.apex.sh/github.com/tj/triage | sh
```

Which would grab a public repo, install the modules, and build the platform's binary on-demand with a cache. Mostly because with dozens or hundreds of projects it'd still be a big pain to configure GH actions, and uploading binaries from a local machine is too slow.

**Comment by caarlos0** [comment]
I'm up to help building such thing FWIW, seems like something fun to work on :D

**Comment by tj** [comment]
Sounds like the Go team is interested in this one, but they said it's lower priority so who knows when

**Comment by tj** [comment]
More or less done, just needs a bit of polish for error reporting and some kind of cool domain name if anyone has ideas :D

**Comment by tj** [comment]
just waiting on blog stuff to announce
